resources/assets/scripts/components/accordion.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="accordion">
+    <a :href="href" @click.prevent="toggle" :id="`${localId}-title`" :aria-controls="`${localId}-content`" :aria-expanded="active" :role="role">
+      <slot name="title"></slot>
+    </a>
+    <div v-show="active" :id="`${localId}-content`" :aria-labelledby="`${localId}-title`" :aria-hidden="!active" :role="role === 'tab' ? 'tabpanel' : null">
+      <slot></slot>
+    </div>
+  </div>
+</template>
+
+<style>
+</style>
+
+<script>
+import { MixinId } from 'vue-a11y-utils';
```

Comment:
This provides `this.localId` which uses `this.id` if available. If not it creates a unique identifier. This is needed for eg. telling screen readers that this specific anchor opens this specific container. If we use `deep-link` option we should set `id` manually as the `localId` gets a new ID every page request.